Makefile.dune

```diff
@@ -59,7 +59,7 @@ voboot:
 	@echo "This target is empty and not needed anymore"
 
 states:
-	dune build --display=short $(DUNEOPT) dev/shim/coqtop-prelude
```

Comment:
This isn't actually necessary for this PR but why do we pass --display here anyway?

Reply:
Because some developers were used to having a verbose log and wanted to see the same with Dune. This `states` target is basically useless except to provide a feeling of similarity for developers used to the previous build system.

Reply:
Why don't they put the option in their global config?

Reply:
I don't know exactly who uses this today. I personally don't and would be fine with removing it.